app/build.gradle

```diff
@@ -264,16 +263,21 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
-
-    // Room - exclude the atomic functions module from the package and prevent warnings.
     packagingOptions {
-        exclude 'META-INF/atomicfu.kotlin_module'
+        pickFirst("META-INF/atomicfu.kotlin_module")
+       /* resources {
+            excludes += ['META-INF/atomicfu.kotlin_module']
+        }*/
     }
+
+    // Room - exclude the atomic functions module from the package and prevent warnings.
+    namespace 'com.concordium.wallet'
+    buildToolsVersion '33.0.1'
 }
 
 dependencies {
     def lifecycle_version = "2.2.0"
-    def room_version = "2.2.5"
+    def room_version = "2.4.3"
```

Comment:
Let's be consistent in using either quotation marks or apostrophes. At the very least in the same file.